src/View/Helper/FormHelper.php

```diff
@@ -1039,6 +1041,40 @@ protected function _ariaOptions(string $fieldName, array $options): array
         return $options;
     }
 
+    /**
+     * Modify options for placeholders.
+     *
+     * @param string $fieldName Field name.
+     * @param array $options Options. See `$options` argument of `control()` method.
```

Comment:
we usually `array<string, mixed>` those
Same for assoc return value.

Reply:
I'd say that's something for a new PR if we want to adpot that syntax, as there's dozens of places that just use the plain `array` hints.

Reply:
Well, new code could already do that, the same as you just added return types :)

Reply:
The other option processing methods already have return types, I only copy pasta'd it 😉